src/backend/PhotoAPI/Domains/DataTransferObjects/PhotoThumbnailDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Domains.DataTransferObjects
+{
+	public class PhotoThumbnailDTO
+	{
+		public System.Guid PhotoId { get; set; }
+
+		public string PhotoUrl64 { get; set; }
+		public string PhotoUrl256 { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5244170)